qa-tests-backend/scripts/migrate_protos.sh

```diff
@@ -9,4 +9,6 @@ cp -r ../proto/* ${JAVA_PATH}
 # Migrate v1 API protos from the Scanner repo
 SCANNER_DIR=$(go list -f '{{.Dir}}' -m github.com/stackrox/scanner)
 # files from gomod cache have no write permission causing problems for gradle
-cp -r --no-preserve=mode ${SCANNER_DIR}/proto/* ${JAVA_PATH}
```

Comment:
So looks like mac has old version https://lists.gnu.org/archive/html/bug-coreutils/2003-02/msg00051.html

Reply:
Thanks for checking.